pkg/apis/clusterlink.net/v1alpha1/import.go

```diff
@@ -48,6 +48,9 @@ type ImportSpec struct {
 	TargetPort uint16 `json:"targetPort,omitempty"`
 	// Sources to import from.
 	Sources []ImportSource `json:"sources"`
+	// The existing service endpoint to merge the imported service with by
+	// creating an endpointslice with the service name pointed to.
+	Merge string
```

Comment:
Also, we may want to treat imports with `merge` set differently (assuming it's going to be the rare case, not norm).
- In most cases (no merge) we create a Service with a Selector pointing to our Pods (in privileged mode this needs to be "redirected" in the import namespace using ExternalName reference to a ClusterLink service in `clusterlink-system` since we can't have cross namespace selectors)
- In case of merge, create an additional EndpointSlice.

Or we choose the EndpointSlice strategy for all cases. @orozery - thoughts?

Reply:
I am changing merge to be a bool and not the import service name. I initially thought it might be needed to keep track of the endpointslice when we perform deletion. But I am realizing we can keep track using a label of the peer.

Upon merge option selected, we would not be creating a service at all right? and only be creating endpointslice.

Reply:
Correct. Only endpoint slice is needed when merging as the service must already exist

Reply:
Need to test if k8s auto deletes all endpoint slices when a service is deleted.

Reply:
No, endpointslice associated managed by us will not be deleted automatically when a service is deleted.

Reply:
But here, endpointslice since it's managed by clusterlink, it shouldn't be deleted unless the user deletes the import right. Otherwise, it might be unintended action taken by it.